### PR TITLE
logging improvements

### PR DESCRIPTION
**Pull request by eonpatapon**

### CODE REVIEW

**Comment by safchain** [comment]
rerun functional-tests-backend-elasticsearch

File: config/config.go [hunk 1]
```diff
@@ -103,6 +103,7 @@ func init() {
 	cfg.SetDefault("logging.encoder", "")
 	cfg.SetDefault("logging.file.path", "/var/log/skydive.log")
 	cfg.SetDefault("logging.level", "INFO")
+	cfg.SetDefault("logging.colorize", true)
```
Comment (safchain) [inline config/config.go]: just color

**Comment by eonpatapon** [comment]
rerun skydive-functional-tests-backend-elasticsearch

**Comment by eonpatapon** [comment]
rerun skydive-functional-tests-backend-orientdb

**Comment by eonpatapon** [comment]
rerun skydive-functional-tests-backend-orientdb

File: config/config.go [hunk 2]
```diff
@@ -103,6 +103,7 @@ func init() {
 	cfg.SetDefault("logging.encoder", "")
 	cfg.SetDefault("logging.file.path", "/var/log/skydive.log")
 	cfg.SetDefault("logging.level", "INFO")
+	cfg.SetDefault("logging.color", true)
```
Comment (nplanel) [inline config/config.go]: keep it sorted in alphabetical order

**Pull request merged**